### ISSUE TITLE
Nontransparent overlay misbehaves on diffrent computer

### ISSUE THREAD

**Issue by Remonell** [caprica/vlcj#776]
Hi,

basically I copied the code of your nontransparent overlay test class. I added a few buttons that change color on click. Also I want to run this in fullscreen. On my dev machine everything works perfect (dedicated Nvidia GPU). On the test machine it starts misbehaving (onboard GPU). 
On the test machine it works if I do not use the software in fullscreen. In fullscreen though, the buttons actionPerformed gets triggered but the buttons do not change the color. If I try to maximize the frame and setUndecorate(true) it sometimes leaves the space that the buttons would occupy and makes it seethrough (can see thorough the mediaplayer).

Probably its not an issue with vlcj or libvlc I assume. But would you please be so kind and share me your thoughts on that matter? It probably hints me in the right direction.  Maybe you have experienced the very same behaviour.

Kind regards

**Comment by caprica** [comment]
How are you doing full-screen?

**Comment by Remonell** [comment]
```
		FullScreenStrategy fss = new Win32FullScreenStrategy(frame);
		mediaPlayer.setFullScreenStrategy(fss);
		mediaPlayer.setFullScreen(false);
```

**Comment by Remonell** [comment]
In addition: In my project I try to open 2 frames on 2 monitors via frame.setLocation(1920,0). This part does not work properly either weirdly. It reads the resolution of both screens properly. But the setLocation does whatever it wants. Very weird.

**Comment by Remonell** [comment]
**SOLVED:** Following 3 lines were the reason for the described behaviour:

```
System.setProperty("sun.awt.noerasebackground", "true");
System.setProperty("sun.awt.erasebackgroundonresize", "false");
System.setProperty("sun.java2d.opengl", "true");
```

After I remove them, my program does what it should do. I am still wondering why it only showed that behaviour on the test machine. Anyways, I hope I can help someone with similar errors. 
Sorry caprica for I have wasted your time. I spent already 15 hours on that specific issue and I was not expecting it to solve it so quickly after posting here.

Kind regards
